lib/DxilCompression/miniz.c

```diff
@@ -2125,10 +2125,11 @@ void *tdefl_write_image_to_png_file_in_memory_ex(const void *pImage, int w, int
         tdefl_output_buffer_putter(&z, 1, &out_buf);
     /* compress image data */
     tdefl_init(pComp, tdefl_output_buffer_putter, &out_buf, s_tdefl_png_num_probes[MZ_MIN(10, level)] | TDEFL_WRITE_ZLIB_HEADER);
-    for (y = 0; y < h; ++y)
-    {
+    for (y = 0; y < h; ++y) {
         tdefl_compress_buffer(pComp, &z, 1, TDEFL_NO_FLUSH);
-        tdefl_compress_buffer(pComp, (mz_uint8 *)pImage + (flip ? (h - 1 - y) : y) * bpl, bpl, TDEFL_NO_FLUSH);
+        tdefl_compress_buffer(
+            pComp, (const mz_uint8 *)pImage + (flip ? (h - 1 - y) : y) * bpl,
```

Comment:
The rest of this file isn't getting clang-formatted because not only does it not belong to DXC, it doesn't belong to LLVM either.  I'm not saying we shouldn't fix the warning here, but I don't think we should clang format the parts you've changed if only to minimize the changes.